lib/pronto/formatter/colorizable.rb

```diff
@@ -0,0 +1,17 @@
+module Pronto
+  module Formatter
+    module Colorizable
+      def colorize(string, color)
+        rainbow.wrap(string).color(color)
+      end
+
+      private
+
+      def rainbow
+        @rainbow ||= Rainbow.new.tap do |rainbow|
+          rainbow.enabled = config.text_formatter_colors && $stdout.tty?
```

Comment:
Much more terse, and more clear, than the [equivalent rubocop cop code](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/formatter/colorizable.rb#L10-L20). Nice job!